src/main.rs

```diff
@@ -29,15 +29,15 @@ mod utils;
 #[derive(Parser)]
 #[clap(author, version, about)]
 struct Args {
-    /// Number of iterations to run.
+    /// Number of seconds to run.
     #[clap(
         long,
         short,
         display_order = 1,
-        default_value_t = 1000,
+        default_value_t = 8,
```

Comment:
I think a better default would be be to run forever (i.e. have _no_ default value for this option).

Reply:
I thought the same but maybe it is more intuitive as a user to get some output after a few seconds without thinking or reading in the documentation how to stop the application ‘Ctrl+C’ to get some results

Reply:
DOne